konflux-ci/integration/core/bootstrap-namespace.yaml

```diff
@@ -20,16 +20,15 @@ spec:
           selector:
             matchLabels:
               konflux-ci.dev/type: tenant
-    celPreconditions:
```

Comment:
@filariow mentioned on slack that the idea behind this was to only trigger the policy when the ns first gets the tenant label. 

With that in mind, I think the condition could be the same as before but replacing the !update with create. In other words, either when the label first applied or when the ns is created.